extensions/ql-vscode/src/vscode-tests/no-workspace/query-history-info.test.ts

```diff
@@ -1,19 +1,18 @@
 import { expect } from 'chai';
```

Comment:
It's unfortunate that this file needs to be moved since the pure tests are much quicker to iterate on when testing changes, but it does seem like the `CompletedQueryInfo` depends on a lot of `vscode`-specific code.

Reply:
Yeah, the feedback loop on running integration tests is too long.